pennylane/beta/queuing/operation.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint: disable=protected-access
+r"""
+This module contains the abstract base classes for defining PennyLane
+operations and observables.
+"""
+
+import pennylane as qml
+
+# --------------------
+# Beta related imports
+# --------------------
+
+from pennylane.operation import Tensor
+
+
+class BetaTensor(Tensor):
```

Comment:
At some point we might want to consider a rename to `TensorProd` since we now have the `qml.numpy.tensor` object?